apps/src/templates/courseOverview/ResourcesEditor.js

```diff
@@ -46,7 +61,13 @@ export default class ResourcesEditor extends Component {
       newResources[index].link = defaultLinks[type];
     }
 
-    this.setState({resources: newResources});
+    let errorString = '';
+    let types = newResources.map(resource => resource.type).filter(resource => resource);
```

Comment:
Tiny, tiny nit: Your arrow function param `.filter(resource => resource)` would be more correctly named `.filter(type => type)`.

Also neat trick for this, though probably too clever by half: `.filter(Boolean)`